src/common/ast-path.js

```diff
@@ -20,6 +21,30 @@ class AstPath {
     this.stack = [value];
   }
 
+  get name() {
+    return getPenultimate(this.stack) || null;
+  }
+
+  get node() {
+    return getLast(this.stack);
```

Comment:
Node.js 16 should already support `.at(-1)` but I guess esbuild only transform new syntax but don't polyfill methods.